app/src/main/assets/mobile-services.json

```diff
@@ -4,47 +4,6 @@
   "namespace": "myproject",
   "clientId": "app-android",
   "services": [
-    {
```

Comment:
Are we removing these services from all the showcase apps?

Reply:
@wei-lee Since services should fail gracefully without a configuration but misconfigured services could cause problems, should we remove the services from other show case's mobile-services?

Reply:
@secondsun yes, I think so